storage/pebble.go

```diff
@@ -0,0 +1,136 @@
+package storage
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/cockroachdb/pebble"
+	"github.com/gnolang/gno/tm2/pkg/bft/types"
+
+	storageErrors "github.com/gnolang/tx-indexer/storage/errors"
+)
+
+var _ Storage = &Pebble{}
+
+// Storage is the instance of an embedded storage
```

Comment:
Nitpick: the comment doesn't match the struct name, probably leftover

Reply:
Resolved in [1a82f41](https://github.com/gnolang/tx-indexer/pull/8/commits/1a82f4175f3bbc267786ab215e4b69760c998958)